maxim/logger/livekit/gemini/gemini_realtime_session.py

```
@@ -0,0 +1,366 @@
+            f"[Internal][Gemini:{self.__class__.__name__}] {hook_name} failed; error={str(e)}\n{traceback.format_exc()}"
+        )
+
+
+def instrument_gemini_session(orig, name):
+    if inspect.iscoroutinefunction(orig):
+
+        async def async_wrapper(self, *args, **kwargs):
+            pre_hook(self, name, args, kwargs)
+            try:
+                result = await orig(self, *args, **kwargs)
+            finally:                
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)